testsuite/Main.hs

```diff
@@ -144,6 +144,8 @@ runClashTest = defaultMain $ clashTestRoot
         , runTest "BlackBoxFunctionHO" def{hdlTargets=[VHDL]}
         , outputTest ("tests" </> "shouldwork" </> "Signal")   allTargets [] [] "BlockRamLazy"       "main"
         , outputTest ("tests" </> "shouldwork" </> "BlackBox") [VHDL]   [] [] "ZeroWidth"          "main"
+        -- as per group "TopEntity" VHDL tests disabled for for Scaffold
+        , runTest "Scaffold" def{hdlTargets=[Verilog],entities=Entities["", "testBench", "mult_prim"]}
```

Comment:
I don't know if you've seen the (relatively) new netlist tests (like output tests but with the netlist AST), but I feel like this feature would potentially benefit from one (as well as this test). See

https://github.com/clash-lang/clash-compiler/blob/master/tests/shouldwork/Netlist/NoDeDup.hs
https://github.com/clash-lang/clash-compiler/blob/master/testsuite/src/Test/Tasty/Clash.hs#L733-L753
https://github.com/clash-lang/clash-compiler/blob/master/clash-lib/src/Clash/Netlist/Types.hs#L149-L156